internal/controllers/clusterextension_controller.go

```diff
@@ -280,7 +280,7 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alp
 	case rukpaksource.StateUnpacked:
 		// TODO: Add finalizer to clean the stored bundles, after https://github.com/operator-framework/rukpak/pull/897
 		// merges.
-		if err := r.Storage.Store(ctx, ext, unpackResult.Bundle); err != nil {
+		if err := r.Storage.Store(ctx, ext.GetName(), unpackResult.Bundle); err != nil {
```

Comment:
Was this just because you realized we only need to store the name and not the whole struct?

Reply:
Yes, the thing we're actually storing is the `unpackResult.Bundle`, and `ext` was passed in purely for its identifier.